web/src/pages/Route/components/Step1/RequestConfigView.tsx

```diff
@@ -388,15 +388,16 @@ const RequestConfigView: React.FC<RouteModule.Step1PassProps> = ({
         name='service_id'
       >
         <Select disabled={disabled}>
-          <Select.Option value=''>{}</Select.Option>
+          {/* NOTE value = '' mean no service_id select, need to find a better way */}
+          <Select.Option value='' key={Math.random().toString(36).substring(7)} >{null}</Select.Option >
```

Comment:
Why use `null` here?

Reply:
`Select.option` need children property and if `{}` and `{null}` is not ok, Which way do you recommend?

Reply:
```tsx
<Option>None</Option>
```

Reply:
if we use`None`, How to distinguish if the service name is None ？

Reply:
1. Too wired to use `None` as Service name in prod;
2. `Select service (optional)`: https://github.com/apache/apisix-dashboard/blob/master/web/src/components/Upstream/UpstreamForm.tsx#L578